.gitignore

```diff
@@ -1,11 +1,97 @@
-/var
-/vendor
+## Directus custom .gitignore
+################################################################################
+
+/dist
+
+# IDE
 /.idea
 /.vscode
-/phpunit.xml
-Thumbs.db
-.DS_Store
+
+# Phpunit
 .phpunit.result.cache
+
+# Php-cs-fixer
 .php_cs.cache
-composer.lock
```

Comment:
We won't commit `composer.lock` since this is now a package and not a full laravel application.

Reply:
Noted !